server/app/build.gradle

```diff
@@ -11,15 +11,21 @@ def apkName = "app-debug-androidTest.apk"
 def serverApkName = "TestServer.apk"
 
 android {
+
+    // > 23, results in build errors:
+    //  package android.test does not exist
```

Comment:
if that's the only problem, then it can be fixed by using androidX test:

https://developer.android.com/training/testing/set-up-project

Reply:
Updated code comment.  I tried using androidx, but it force the midSdkVersion to be at least 14